caom2utils/caom2utils/caom2blueprint.py

```diff
@@ -4307,7 +4601,7 @@ def __init__(self, header, file, extension):
         :param extension: which HDU
         WCS axes methods of this class.
         """
-        super().__init__()
+        self.logger = logging.getLogger(self.__class__.__name__)
```

Comment:
Just checking that the removal of `super().__init__()` was intentional.

Reply:
The removal is intentional.

The `WcsParser` and `Hdf5WcsParser` manually build up the the `Wcsprm` data structure, where-as for the `FitsWcsParser`, `astropy` already does that from a FITS header.

If you're aware of HDF5 support for WCS construction in `astropy`, let me know, and I'll use it instead of my own code.